app/api/entities/entities.js

```diff
@@ -534,8 +535,17 @@ export default {
   },
 
   async saveMultiple(docs) {
-    await docs.reduce(async (prev, doc) => {
+    const templateIds = Array.from(new Set(docs.map(d => d.template)));
+    const indexedTemplates = objectIndex(
+      await templates.get({ _id: { $in: templateIds } }),
+      t => t._id.toString(),
+      t => t
+    );
+
+    await docs.reduce(async (prev, _doc) => {
       await prev;
+      const template = indexedTemplates[_doc.template];
+      const doc = this.sanitize(_doc, template);
```

Comment:
Just to understand: this was the only change needed because `sanitize` already set's empty as arrays, but we were missing it in `saveMultiple` and to fix the db?

Reply:
Nice work!